### ISSUE TITLE
Remove repository

### ISSUE THREAD

**Issue by codeHusky** [Christian-Schiffer/SignalRAngularMVC6#6]
This repository is using precious GitHub disk space. In order to be courteous of your fellow software developers, I request that you remove this "original content" and clear space on GitHub's servers.

Thank you.

**Comment by Christian-Schiffer** [comment]
> This repository is using precious GitHub disk space. In order to be courteous of your fellow software developers, I request that you remove this "original content" and clear space on GitHub's servers.
> 
> Thank you.

I understand this is yet another spam comment related to me opening issue https://github.com/microsoft/vscode/issues/87268 on the santa hat. If these kinds of spam comments and bad jokes were all that it was, we could all have a good laugh together. Yet, even if that were the case, the sheer magnitude of such spam alone borders to horracement. Unfortunately, spammy issue openings and comments like yours were pretty much the best of them.

Responses ranging from racist name calling including n and k words, forcing your religion down my throat by spamming my repositories and personal email account with santa's, Jesus loves you and merry xmas comment, DDOS attacks on my dev server attempts to hack my dev server, registering with my email on diverse porn and sex and gay dating sites as well as spammer networks, opening fake issues and sending emails to my personal account with disgusting photos such as a santa making the Hitler Salute even one of the github users sending me an email showing his rectum, opened wide, mind you, Hitler salutes all the way to suggestion I should go and kill myself as well as outright death threats and incitement to genocide. And then on top of it all, opening issues with complaint about me because of those incitements against me, people of color, jews and so force.

One would think that developers at the very least would have some deductive reasoning, so using the n and k word against me, when my profile picture and name clearly show I am neither, I am just a dev who is trying to do the right thing and my complaint about that religious symbolism on globally available software that has nothing to do with religion whatsoever is completely out of place as well as pointing out that the particular religious symbol, the santa hat hat been used in xmas traditions over the centuries to mock, tarnish, murder and torture people, especially Jews thus for many is a frightening symbol almost as bad as the swastika. I also added a link to document this. Microsoft of course understood that their well meant holiday greeting has diverse effects on diverse population and ethnic groups and that some of us do not feel particularly happy about getting such symbols forced down our throats and that some even experience fear and anxiety due to the historic meaning of said santa hat.

Now, I was not surprised that not everyone agreed with my concern, yet given I felt I explained why and documented it well, I would have expected supposedly intelligent fellow developers to show understanding and respect for my position. Now a friendly little tarnish against me here and there, while not necessarily nice or even appropriate, I would have accepted and not even bothered to comment upon, much less send abuse complaints about, but this aforementioned hatefest that came in the aftermath, isa not only completely unacceptable, it's not only criminal, some of the offences are outright crimes against humanity, especially those containing incitement to genocide against a protected group. Thus I obviously will have to report some of this to the proper authorities. So those who think they can get away with trying to hack my dev server, launching dos or ddos attacks against said server, utter death threats against my person and even whole ethnic groups including crimes against humanity like inciting to genocide, unpunished, think again. Those issues I will cooperate with Microsoft on and report to the proper authorities.

The aforementioned hates fest by the way also proves the absolute necessity to keep dev apps and networks as well other professional apps and networks free of religious symbolism, apparently especially xmas related symbolism since history in fact did repeat itself and I for some reason became a proxy for the hate these people have for Black people and Jews. I can only imagine how those groups must feel, I after all have only been subject to this for about 48 hours, the groups in question however have to live with this their whole lives. So think people and stop this madness. It most certainly is not funny, not funny at all.

Your fake opening of issue of course excluded, I won't even report it to GitHub, as I said, yours probably represents the least offensive activity directed against me and aforementioned ethnic group who have suffered greatly because of such incitement. Yet I would urge you to apologize for taking part in said hatefest and also the next time, think about what it is you associate yourself with. Thank you.

**Comment by vojtechkral** [comment]
> Now, I was not surprised that not everyone agreed with my concern, yet given I felt I explained why and documented it well

I disagree that you documented it well. The article you linked is sketchy at best and in my opinion in some spots contains downright falsehoods. One example is:

> Christmas celebrates the birth of Jesus, a Christian god who came to rescue mankind from the “curse of the Torah.”

This is absolutely _not_ the way the majority of christians and mainstream denominations understand Jesus. As a matter of fact, mainstream christians believe the exact opposite and in [Matthew 5:17](https://biblehub.com/matthew/5-17.htm) Jesus pretty much says so himself.

The article contains a number of other inaccuracies, simplifications and falsehoods with a pretty apparent purpose to render christianity-to-judaism relationship in the worst possible light. The article itself could very easily be described as offensive. Comparing Christmas to "Hitlerday" really is _not_ exactly sensitive towards christians.

In my opinion your interpretation of christmas and your issue in VSCode repository is a completely needless effort to build fences between groups which are in no way necessarily in opposition. In my opinion it's a promotion of needless animosity between people.

Despite exaggerations in the linked article, I _do_ recognize that the behaviour of christians towards jews was not exactly great at various points in history. There's generally little doubt about that. However, it's virtually impossible to find a religious or cultural group that has not sustained unfair treatment and damange from some other group or groups throughout history. It's questionable whether diggin up grievances from centuries or thousands of years ago serves any useful purpose today besides reinforcing divisions between cultural groups.

Please note tht I do not approve of the treatment you've received since opening the VSCode issue in any way.

In summary: Please reconsider your position. A christmas hat is simply a festivity, nothing else, and the _vast_ majority of people celebrating christmas do not perceive any antagonism against the Torah by celebrating christmas (there's really no reason for that). In fact, as far as I know, most of them feel the opposite way.

cc. @kieferrm @egamma : Please also reconsider.

**Comment by nickolaym** [comment]
Christian, you see, your grandpa had a choice to be jailed or to become a nazi. He selected. He offended (and killed) lots of people. And he paid a known cost for that.
Then, you had a choice to celebrate most funny holiday, or just ignore, or go to war against majority of people. You have selected. You do offend people. Now pay.

BTW, your act put a very dark shade on all hebrews "20 centuries ago they crucified God, and now they enforce us stop celebrating His birthday". Thank you very much, from all hebrews.

And maybe you indeed is not a hebrew? Maybe you are neonazi like your grandpa, and do that specially to embroil christians against hebrews?

**Comment by nallg00d** [comment]
> > This repository is using precious GitHub disk space. In order to be courteous of your fellow software developers, I request that you remove this "original content" and clear space on GitHub's servers.
> > Thank you.
> 
> I understand this is yet another spam comment related to me opening issue [microsoft/vscode#87268](https://github.com/microsoft/vscode/issues/87268) on the santa hat. If these kinds of spam comments and bad jokes were all that it was, we could all have a good laugh together. Yet, even if that were the case, the sheer magnitude of such spam alone borders to horracement. Unfortunately, spammy issue openings and comments like yours were pretty much the best of them.
> 
> Responses ranging from racist name calling including n and k words, forcing your religion down my throat by spamming my repositories and personal email account with santa's, Jesus loves you and merry xmas comment, DDOS attacks on my dev server attempts to hack my dev server, registering with my email on diverse porn and sex and gay dating sites as well as spammer networks, opening fake issues and sending emails to my personal account with disgusting photos such as a santa making the Hitler Salute even one of the github users sending me an email showing his rectum, opened wide, mind you, Hitler salutes all the way to suggestion I should go and kill myself as well as outright death threats and incitement to genocide. And then on top of it all, opening issues with complaint about me practically claiming I forced them to utter those incitements against me, people of color and Jews by opening issue #87268 . I mean c'mon, are these people for real?
> 
> One would think that developers at the very least would have some deductive reasoning, so using the n and k word against me, when my profile picture and name clearly show I am neither, I am just a dev who is trying to do the right thing and my complaint about that religious symbolism on globally available software that has nothing to do with religion whatsoever is completely out of place as well as pointing out that the particular religious symbol, the santa hat hat been used in xmas traditions over the centuries to mock, tarnish, murder and torture people, especially Jews thus for many is a frightening symbol almost as bad as the swastika. I also added a link to document this. Microsoft of course understood that their well meant holiday greeting has diverse effects on diverse population and ethnic groups and that some of us do not feel particularly happy about getting such symbols forced down our throats and that some even experience fear and anxiety due to the historic meaning of xmas including its symbolism as said santa hat.
> 
> Now, I was not surprised that not everyone agreed with my concern, yet given I felt I explained why and documented it well, I would have expected supposedly intelligent fellow developers to show understanding and respect for my position. Now a friendly little tarnish against me here and there, while not necessarily nice or even appropriate in this context, I would have accepted and not even bothered to comment upon, much less send abuse complaints about, but this aforementioned hatefest that came in the aftermath, is not only completely unacceptable, in fact not only regular criminal offences, some of the offences are outright crimes against humanity, especially those containing incitement to genocide against a protected group. Thus I obviously will have to report some of this to the proper authorities. So those who think they can get away with trying to hack my dev server, launching dos or ddos attacks against said server, utter death threats against my person and even whole ethnic groups including crimes against humanity like inciting to genocide, unpunished, think again. Those issues I will cooperate with Microsoft on and assist in reporting to the proper authorities.
> 
> The aforementioned hates fest by the way also proves the absolute necessity to keep dev apps and networks as well other professional apps and networks free of religious symbolism, apparently especially xmas related symbolism since history in fact did repeat itself and I for some reason became a proxy for the hate these people have for Black people and Jews. I can only imagine how those groups must feel, I after all have only been subject to this for about 48 hours, the groups in question however have to live with this their whole lives. So think people and stop this madness. It most certainly is not funny, not funny at all.
> 
> Your fake opening of issue of course excluded, I won't even report it to GitHub, as I said, yours probably represents the least offensive activity directed against me and aforementioned ethnic group who have suffered greatly because of such incitement. Yet I would urge you to apologize for taking part in said hatefest and also the next time, think about what it is you associate yourself with. Thank you.
> 
> _See also the closed issues and pull requests as well as the comments on my repository servicelayer/chat which I had to lock down for the time being because of this as they will give you some clues about what has been going on since I opened issue #87268 ._

Are you really getting death threats and all that stuff you said over this??

**Comment by Pizzacus** [comment]
@nallg00d As I understand, 4chan was involved, and as you know, they like to take things to the extreme...

**Comment by Enerccio** [comment]
> > This repository is using precious GitHub disk space. In order to be courteous of your fellow software developers, I request that you remove this "original content" and clear space on GitHub's servers.
> > Thank you.
> 
> I understand this is yet another spam comment related to me opening issue [microsoft/vscode#87268](https://github.com/microsoft/vscode/issues/87268) on the santa hat. If these kinds of spam comments and bad jokes were all that it was, we could all have a good laugh together. Yet, even if that were the case, the sheer magnitude of such spam alone borders to horracement. Unfortunately, spammy issue openings and comments like yours were pretty much the best of them.
> 
> Responses ranging from racist name calling including n and k words, forcing your religion down my throat by spamming my repositories and personal email account with santa's, Jesus loves you and merry xmas comment, DDOS attacks on my dev server attempts to hack my dev server, registering with my email on diverse porn and sex and gay dating sites as well as spammer networks, opening fake issues and sending emails to my personal account with disgusting photos such as a santa making the Hitler Salute even one of the github users sending me an email showing his rectum, opened wide, mind you, Hitler salutes all the way to suggestion I should go and kill myself as well as outright death threats and incitement to genocide. And then on top of it all, opening issues with complaint about me practically claiming I forced them to utter those incitements against me, people of color and Jews by opening issue #87268 . I mean c'mon, are these people for real?
> 
> One would think that developers at the very least would have some deductive reasoning, so using the n and k word against me, when my profile picture and name clearly show I am neither, I am just a dev who is trying to do the right thing and my complaint about that religious symbolism on globally available software that has nothing to do with religion whatsoever is completely out of place as well as pointing out that the particular religious symbol, the santa hat hat been used in xmas traditions over the centuries to mock, tarnish, murder and torture people, especially Jews thus for many is a frightening symbol almost as bad as the swastika. I also added a link to document this. Microsoft of course understood that their well meant holiday greeting has diverse effects on diverse population and ethnic groups and that some of us do not feel particularly happy about getting such symbols forced down our throats and that some even experience fear and anxiety due to the historic meaning of xmas including its symbolism as said santa hat.
> 
> Now, I was not surprised that not everyone agreed with my concern, yet given I felt I explained why and documented it well, I would have expected supposedly intelligent fellow developers to show understanding and respect for my position. Now a friendly little tarnish against me here and there, while not necessarily nice or even appropriate in this context, I would have accepted and not even bothered to comment upon, much less send abuse complaints about, but this aforementioned hatefest that came in the aftermath, is not only completely unacceptable, in fact not only regular criminal offences, some of the offences are outright crimes against humanity, especially those containing incitement to genocide against a protected group. Thus I obviously will have to report some of this to the proper authorities. So those who think they can get away with trying to hack my dev server, launching dos or ddos attacks against said server, utter death threats against my person and even whole ethnic groups including crimes against humanity like inciting to genocide, unpunished, think again. Those issues I will cooperate with Microsoft on and assist in reporting to the proper authorities.
> 
> The aforementioned hates fest by the way also proves the absolute necessity to keep dev apps and networks as well other professional apps and networks free of religious symbolism, apparently especially xmas related symbolism since history in fact did repeat itself and I for some reason became a proxy for the hate these people have for Black people and Jews. I can only imagine how those groups must feel, I after all have only been subject to this for about 48 hours, the groups in question however have to live with this their whole lives. So think people and stop this madness. It most certainly is not funny, not funny at all.
> 
> Your fake opening of issue of course excluded, I won't even report it to GitHub, as I said, yours probably represents the least offensive activity directed against me and aforementioned ethnic group who have suffered greatly because of such incitement. Yet I would urge you to apologize for taking part in said hatefest and also the next time, think about what it is you associate yourself with. Thank you.
> 
> _See also the closed issues and pull requests as well as the comments on my repository servicelayer/chat which I had to lock down for the time being because of this as they will give you some clues about what has been going on since I opened issue #87268 ._

can you tell me how does santa hat relate to christianity at all?

**Comment by dreamsComeTrue** [comment]
if you don't like little hat - JUST DON'T USE IT ANYMORE!!! Don't brek Christmas, the time of joy of our Lord's birth and just don't force anyone to go with your beliefs. Market is free, there are other choices of code editors

**Comment by blockedbychristianschiffer** [comment]
My main account was blocked even though an apology was asked for. Since this comes off as an attempt to make me look like an ass, you won't be earning your apology.

 I understand if you hold beliefs that don't align with other people, and I'll even let some of your blatant false information slide. However, this is a formal request to either 
a) Leave the tech industry and seek out another profession more appropriate to your interests, such as the study of religion. (It might be very enlightening for you)
b) Respond to the problem by apologizing for demonizing a part of an ad created by Coca-Cola. 

Now, I'm not a Coca-Cola employee, but pressuring Microsoft into removing a graphic from their software under the pretense of some kind of offense (while being completely wrong in your background information on the topic) violates my American values. There are many projects which actually try to stir up hate against the Jewish people across GitHub which honestly require your attention more than VSCode. If you applied your same pursursuasive abilities in the form of a GitHub abuse email, I'm sure you'd be very successful in stopping actual discrimination against the Jewish people. Additionally, spending some of your money donating to organizations that help Holocaust survivors would be another good step, as I'm not sure why you wouldn't. After all, it's the nice thing to do, and I'd imagine you're working full time. I'm sure you can spare a dollar or two.

Please, comb through your replies in this mess of hatred. There are some legitimately rational people down there (and in other issues) trying to explain to you why you're completely misinformed, they just can't do it without calling you names. At the same time, if you're not willing to change, frankly I wouldn't mind if you left the industry. It's not out of hatred of you, it's just for the sake of work environments everywhere.

**Comment by nallg00d** [comment]
> @nallg00d As I understand, 4chan was involved, and as you know, they like to take things to the extreme...

ugh..

**Comment by gryter** [comment]
You ruined something people enjoyed while claiming to represent an entire group of people. You knew full-well that Microsoft is too afraid of triggering a crazy debate about hurt feelings. You also needlessly involved the Jews, while most of us who are of that ethnicity respect and celebrate all holidays. This is unprofessional and brought your personal politics to GitHub. It doesn't warrant threats but it does warrant a discussion about why political goals should not be achieved through GitHub. For obvious reasons that inspired all of us to come here to see what kind of non-offensive projects you are involved with.

**Comment by Rexogamer** [comment]
May I state that you wanted people to stop bringing personal politics and religion into this website, yet this whole mess started because you did that very thing?

**Comment by Christian-Schiffer** [comment]
> if you don't like little hat - JUST DON'T USE IT ANYMORE!!! Don't brek Christmas, the time of joy of our Lord's birth and just don't force anyone to go with your beliefs. Market is free, there are other choices of code editors

I am not forcing my believes down anyone throat, on the contrary, I am asking you not to force yours down my throat. Visual Studio Code is a product for user globally, not everyone shares your faith or whatever you call it. Please keep your religion to yourself. Thank you.
